modules/transactions/src/main/java/org/apache/ignite/internal/tx/impl/TxManagerImpl.java

```diff
@@ -57,7 +65,12 @@
  *
  * <p>Uses 2PC for atomic commitment and 2PL for concurrency control.
  */
-public class TxManagerImpl implements TxManager {
+public class TxManagerImpl implements TxManager, NetworkMessageHandler {
```

Comment:
I don't see where TxManager is used as a NetworkMessageHandler. Also, I do not understand why it is necessary to calculate inflight operations for the algorithm.

Reply:
Here org.apache.ignite.internal.tx.impl.TxManagerImpl#start
Inflight calculation is required for safe committing a transaction, see ticket description.